src/main/java/calebyyy/Ui.java

```diff
@@ -0,0 +1,81 @@
+package calebyyy;
+import java.util.ArrayList;
+
+import calebyyy.tasks.Task;
+
+/**
+ * Represents the user interface of the application.
+ */
+public class Ui {
+    /**
+     * Constructor for Ui.
+     */
+    public Ui() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Calebyyy");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Print task added message.
+     * @param task
+     * @param taskCount
+     */
+    public void addTaskMessage(Task task, int taskCount) {
+        System.out.println("____________________________________________________________");
```

Comment:
Since you are printing the seperator so many times, could consider putting it into a static string